src/models/Review.py

```diff
@@ -0,0 +1,31 @@
+#temp
+from bson import ObjectId
+import datetime
+
+class Review:
+  _id: ObjectId
+  user_id: str
+  drink_id: str
+  comment: set[str] = set() #comments of drinks from this review
```

Comment:
I think the `comment` should be a singular string. Would there be a use case for a `Review` to store multiple comments?